### ISSUE TITLE
Styling for jsxc v4.0 of Button with div class jsxc-mam-load-more jsxc-show is white on white

### ISSUE THREAD

**Issue by tigexplorer** [mwalbeck/nextcloud-breeze-dark#124]
**Describe the bug**
Button in the div with class `jsxc-mam-load-more jsxc-show` is white on white

**To Reproduce**
1. open a JSXC message box
2. It may appear `Ältere Nachrichten laden (load old messages)`
3. this Button is white text of white background

**Expected behavior**
It may styled like other buttons on dark background, here in the message box

**Screenshots**

![JSXC_Message_Box](https://user-images.githubusercontent.com/43775552/80863707-636bd280-8c7e-11ea-9389-1508da695686.png)

**Setup (please complete the following information):**
 - Nextcloud version: [18.0.3]
 - Theme version [18.0.13]
 - Browser and browser version [Firefox 75.0]

**Comment by mwalbeck** [comment]
Hi,

Thanks for reporting this,

There are a few more issues related to jsxc that I'm working on over here #113 . Though I've run into a bit of issue maybe you can help with. I currently don't have an xmpp server and I had a few problems registering for a managed one that jsxc themselves provide. The reason that is important, is that this issue isn't relevant on the internal xmpp server. The is also the case for any issues with muc's currently as that isn't supported by the internal xmpp server. So thank you for provide the div class, it help with finding the correct rules to change in the original jsxc stylesheet.

Now, could I get you to download the Custom CSS app for nextcloud and add the styling below to see if that fixes it.
```
.jsxc-window .jsxc-mam-load-more span {
    background-color: var(--color-main-background);
}

.jsxc-window .jsxc-mam-load-more span:hover {
    background-color: var(--color-main-background);
    color: var(--color-text-active);
}
```

**Comment by mwalbeck** [comment]
The registration for a managed server went through this time, so I can handle the rest on my own. The fix will be in 18.0.14, probably out tomorrow.

**Comment by tigexplorer** [comment]
Thanx for the quick fix, it works!
